x-pack/plugins/reporting/tsconfig.json

```diff
@@ -42,5 +42,10 @@
   ],
   "exclude": [
     "target/**/*",
+    "x-pack/plugins/reporting/server/routes/diagnostic/screenshot.ts",
+    "x-pack/plugins/reporting/server/routes/generate/csv_searchsource_immediate.ts",
+    "x-pack/plugins/reporting/server/export_types/csv_searchsource_immediate/*",
+    "x-pack/plugins/reporting/server/export_types/png_*/*",
+    "x-pack/plugins/reporting/server/export_types/printable_pdf/execute_job/*",
```

Comment:
let's reset this back to the code from `main` as well

`git checkout main x-pack/plugins/reporting/tsconfig.json`